python/GafferUI/LayoutPlugValueWidget.py

```diff
@@ -43,6 +43,9 @@ class LayoutPlugValueWidget( GafferUI.PlugValueWidget ) :
 
 	def __init__( self, plug, **kw ) :
 
+		if isinstance( plug, set ) :
+			raise GafferUI.PlugValueWidget.MultiplePlugsError()
```

Comment:
This makes it easier for widget creating code to gracefully fail with 'legacy' widgets.